src/components/tipRecords/TipRecord.vue

```diff
@@ -98,13 +98,13 @@ export default {
       return this.tip.preview.responseUrl ? new URL(this.tip.preview.responseUrl).hostname : '';
     },
     tipPreviewImage() {
-      return this.isPreviewToBeVisualized(this.tip) ? Backend.getTipPreviewUrl(this.tip.preview.image) : '';
+      return this.isPreviewToBeVisualized(this.tip) && this.tip.preview.image !== null ? Backend.getTipPreviewUrl(this.tip.preview.image) : '';
     },
   },
   methods: {
     isPreviewToBeVisualized(tip) {
       return typeof tip !== 'undefined' && tip !== null
-          && typeof tip.preview !== 'undefined' && tip.preview.image !== null
+          && typeof tip.preview !== 'undefined'
```

Comment:
why does it make a change where this check is defined?

Reply:
I guess preview could be without image?

Reply:
I see

Reply:
This prevents the fallback image from showing.